src/autoloads/characters.gd

```diff
@@ -80,6 +91,17 @@ func getCharacterResource(id: int) -> CharacterResource:
 		# if running in release mode, return null
 		return null
 	return _characterResources[id]
+	
+func removeCharacterResource(id: int) -> void:
+	# if there is no character node corresponding to this network id
+	if not id in _characterResources:
+		# throw an error
+		printerr("Trying to get a nonexistant character resource with network id ", id)
+		# crash the game (if running in debug mode) to assist with debugging
+		assert(false, "Should be unreachable")
+		# if running in release mode, return
+		return
+	_characterResources.erase(id)
```

Comment:
same question as above.

Reply:
seems to me that Resources inherit from the "Reference" class, so, ", resources are reference-counted and freed when no longer in use", which, I'd guess means that they can be left alone, it is enough to erase the entry from the dictionary.